config/defaults.ini

```diff
@@ -244,6 +244,11 @@ automatic_provision_superuser_name =
 automatic_provision_superuser_password =
 automatic_provision_preset = nonformal
 automatic_provision_landing_page = learn
+automatic_provision_learner_can_edit_username = false
+automatic_provision_learner_can_edit_name = false
+automatic_provision_learner_can_edit_password = false
+automatic_provision_learner_can_sign_up = false
+automatic_provision_allow_guest_access = false
```

Comment:
I'll be happy with this if we can keep these settings as they are upstream (_unset_ if it is possible?). Instead, something like `learner_can_edit_name = false` should be in the configuration for a specific image.

Reply:
Yeah good point. It's a bit of a pain to build a JSON from Bash, but you are right.

Reply:
After a second though I tend to disagree, see my comment in phabricator. https://phabricator.endlessm.com/T32885#919180

Reply:
One thing we could do is decuple the device "can guests access" setting from the rest which are valid in all cases.